aiida_common_workflows/workflows/eos.py

```diff
@@ -46,6 +46,12 @@ def validate_scale_increment(value, _):
         return 'scale increment needs to be between 0 and 1.'
 
 
+def validate_relax(value, _):
+    """Validate the `generator_inputs.relax_type` input."""
+    if 'CELL' in value.name or 'VOLUME' in value.name:
```

Comment:
I realize you are only moving this code, but this we should change for checking against actual `RelaxType` instances. Whole idea of using an enum is to not have to rely on arbitrary string values.
```suggestion
    if value not in [RelaxType.NONE, RelaxType.ATOMS, RelaxType.SHAPE, RelaxType.ATOMS_SHAPE]:
```